automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/library/DataLibrarySpec.scala

```diff
@@ -51,4 +52,50 @@ class DataLibrarySpec extends FreeSpec with WebBrowserSpec with UserFixtures wit
       }
     }
   }
+
+  "For a dataset with facets" in withWebDriver { implicit driver =>
+    val curatorUser = UserPool.chooseCurator
```

Comment:
I'm not exactly sure what this test is trying to verify. that the values show up in the facets section? The names of the tests above might also be confusing, but they are verifying that the consent codes and tags show up in the table correctly. Maybe just a more specific name for the test like "... values are options for facet search"